### PR TITLE
Base: Add which(1) and stat(1) manpages

### PR DESCRIPTION
**Pull request by jahway603**
Added manpages for which and stat to work on Issue https://github.com/SerenityOS/serenity/issues/11311 in the spirit of #Hacktoberfest2022

### CODE REVIEW

File: Base/usr/share/man/man1/which.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Name
+
+which - shows the full path of commands
+
+## Synopsis
+
+```**sh
```
Comment (networkException) [inline Base/usr/share/man/man1/which.md:7]: What are these asterisks for?

**Comment by networkException** [comment]
Sorry, those characters are actually supposed to be there. Although I'm not quite sure why there are only two in the beginning. The parser comment says otherwise: https://github.com/SerenityOS/serenity/blob/cede752cd1ebad45a1559048cdb1d59e5a3c2fdf/Userland/Libraries/LibMarkdown/CodeBlock.cpp#L131

This is nothing you have to worry about though, staying consistent with other man pages is good for now

**Comment by jahway603** [comment]
@networkException so I should fix both manpages with two asterisks before and after sh but under Synopsis only and not under Examples?

**Review by davidot — CHANGES_REQUESTED** [review]
> Also, I'm not sure how to fix the linter issues that @BuggieBot states are a problem.

The commit messages are not as they are defined in https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy .
They should for example be:
`Base: Add manpage for which`

Also can you please drop or squash the 3rd commit you added? 
I'm not sure if that change should be kept check networkExceptions comment for that

**Comment by jahway603** [comment]
@davidot I understand now that the commit message has to be `Base: Added manpage for xyz`, but I'm not sure how to change the commit messages now. I also do not know how to drop or squash the 3rd commit & wonder if I need to change these manpages as @networkException suggested. Can you please help me learn how to do this here? Thank you.

**Comment by jahway603** [comment]
@davidot this is a cool video and I watched it. After I merged to one commit locally, it won't let me push so had to create a new branch and PR https://github.com/SerenityOS/serenity/pull/15747 to make it work. I might be missing something fundamental.

**Comment by networkException** [comment]
Assuming you had the branch checked out locally `git push --force` should work, but lets not complicate this further and continue in #15747

**Comment by jahway603** [comment]
@networkException thank you for that. I'll have to explore all this git merge and more with a blank repo.

**Pull request closed without merging**